recipes/maxsmi/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "maxsmi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/volkamerlab/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: 3c936f9486cf05e1a73c1b10815510ad1d5f94073998aab3bd72be59f9902de1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - jupyter
+    - jupyterlab
+    - rdkit >=2018.09.2.0
+    - scikit-learn
+    - requests
+    - tqdm
+    - pytorch
+    - matplotlib
```

You probably want just `matplotlib-base` here to save the `qt` download (you are only using matplotlib in a notebook context, right?).

```suggestion
    - matplotlib-base
```